tsl/src/chunk.c

```diff
@@ -1044,6 +1086,14 @@ chunk_merge_chunks(PG_FUNCTION_ARGS)
 
 			if (mergeindex == -1)
 				mergeindex = i;
+
+			/* Read compression chunk size stats */
+			bool found = ts_compression_chunk_size_get(chunk->fd.id, &relinfo->ccs);
+
+			if (!found)
+				elog(NOTICE,
```

Comment:
Notice is something that goes to psql, right? I'm not sure what user action we expect based on this. Maybe it should be a warning or just a log.